package.json

```diff
@@ -128,7 +128,7 @@
     "babel-plugin-ember-template-compilation": "^2.1.1",
     "dag-map": "^2.0.2",
     "decorator-transforms": "2.0.0",
-    "ember-cli": "^4.10.0",
+    "ember-cli": "^5.12.0",
```

Comment:
This bump is needed because the .gts -> .gjs conversion is only available since ember-cli 5.5. I took 5.12 since it's the current LTS, but I can switch to a different version if needed.

Reply:
It seems this bump also causes other test failures related to pods, need to look into this further but the failures are unrelated to the new code here.

Reply:
I worked around it by using the `--pod` argument where needed instead. Most pod tests already did that, only a specific set of tests where using the `usePods` option.